service/models.py

```diff
@@ -129,7 +152,7 @@ class OrderItem(db.Model):
     ##################################################
     id = db.Column(db.Integer, primary_key=True)
     quantity = db.Column(db.Integer)
-    order_id = db.Column(db.Integer)
+    order_id = db.Column(db.Integer, db.ForeignKey("Order.id"), nullable=False)
```

Comment:
Please add ``ondelete="CASCADE"`` (or another explicit rule such as ``RESTRICT``) to the order_id foreign-key.
With ON DELETE CASCADE the database removes child rows in one pass!

```
order_items = db.relationship(
    "OrderItem",
    backref="order",
    cascade="all, delete-orphan",
    passive_deletes=True
)
```